internal/common/conversion/type_conversion_test.go

```diff
@@ -67,3 +68,31 @@ func TestMongoDBRegionToAWSRegion(t *testing.T) {
 		}
 	}
 }
+
+func TestSlice_SameBehavior(t *testing.T) {
+	tests := []struct {
+		ptr   *[]string
+		slice []string
+	}{
+		{nil, nil},
+		{&[]string{"hello", "there"}, []string{"hello", "there"}},
+	}
+	for _, test := range tests {
+		assert.Equal(t, test.slice, conversion.SlicePtrToSlice(test.ptr))
+		assert.Equal(t, test.ptr, conversion.NonEmptySliceToSlicePtr(test.slice))
+	}
```

Comment:
when doing test matrix please use `t.Run` for the test case

Reply:
thx, done